ntbs-service/Pages/Notifications/Edit/Episode.cshtml.cs

```diff
@@ -78,6 +89,10 @@ protected override IActionResult RedirectToNextPage(int? notificationId, bool is
         protected override async Task ValidateAndSave()
         {
             SetValuesForValidation();
+            if(Notification.NotificationStatus != Models.Enums.NotificationStatus.Draft && Notification.Episode.TBServiceCode != Episode.TBServiceCode)
+            {
+                ModelState.AddModelError("Episode.TBServiceCode", ValidationMessages.TBServiceCantChange);
```

Comment:
manually adding the modelState error here if the TB service changes relative to the Notification loaded from the database - will add a test for this